### PR TITLE
docs: readme

### PR DESCRIPTION
**Pull request by j-zimnowoda**
#464

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,123 +4,32 @@
...
-
-After changing code you can do a diff to see everything still works and what has changed in the output manifests:
-
-```bash
-otomi diff
-# or target one release:
-otomi diff -l name=prometheus-operator
+helm repo add otomi https://otomi.io/otomi-core
+helm repo update
+helm install otomi otomi/otomi
```
Comment (mojtabaimani) [inline README.md:22]: They  can not install otomi chart without providing values. so it should be `helm install -f values.yaml otomi otomi/otomi`

**Review by mojtabaimani — CHANGES_REQUESTED** [review]

File: README.md [hunk 2]
```diff
@@ -4,123 +4,32 @@
...
+Otomi Core is the heart of the Otomi Container Platform, which extends Kubernetes with an advanced ingress architecture, curated industry proven applications and policies for better governance and security.
 
-Important features:
+Otomi aims to support the most common DevOps use cases and strongly relies on GitOps pattern, where desired state is reflected as code and any change to that manifest is reflected at the platform.
```
Comment (Morriz) [inline README.md:9]: I don't think that is really what people perceive as gitops, and maybe we should rename it to ConfOps. GitOps is described as code being synced from git, but we have all code in containers and only conf is left in git.

Anyways, this line does not belong here in this list so please remove it here. Maybe you want to add a small section below on ConfOps

Comment (Morriz) [reply README.md:9]:
```suggestion
```

**Review by Morriz — COMMENTED** [review]
Thanks for the attempt to create less text. For now I wish to close this PR, because I want to write it myself.

**Comment by j-zimnowoda** [comment]
> Thanks for the attempt to create less text. For now I wish to close this PR, because I want to write it myself.

ok, but I thought we put things to sprint column, so people can assign themselves. If you knew from the very beginning that you want to perform this story then you should have assigned yourself in advance, otherwise you waste someones time.

File: README.md (hunk 2 above)
Comment (j-zimnowoda) [reply README.md:13]: Maybe I can describe it as configuration as code that follows GitOps pattern

File: .cspell.json [hunk 3]
```diff
@@ -20,6 +20,7 @@
     "gotmpl",
     "groupsname",
     "gzrs",
+    "Hashi",
```
Comment (svatwork) [inline .cspell.json:23]:
```suggestion
    "HashiCorp",
```

**Review by svatwork — CHANGES_REQUESTED** [review]

**Comment by Morriz** [comment]
closing as it is not following the format we have in mind

**Pull request closed without merging**